.github/workflows/flutter-ci.yml

```diff
@@ -0,0 +1,39 @@
+name: Flutter CI
+
+on: pull_request
+
+jobs:
+  build:
+    runs-on: macos-latest
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v1
+
+      - name: Setup Java
+        uses: actions/setup-java@v1
+        with:
+          java-version: '12.x'
+
+      - name: Setup Flutter
+        uses: subosito/flutter-action@v1
+        with:
+          channel: 'stable'
```

Comment:
I found that you can replace these both setup actions by using Google's Dart container:

```
container:
      image: google/dart:2.12
```

References:
https://github.com/felangel/bloc/blob/master/.github/workflows/bloc.yaml
https://www.bradcypert.com/dart-packages-with-github-actions/